src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderSentence.kt

```diff
@@ -23,9 +29,27 @@ data class SentenceId(
 
 @Entity
 @Table(name = "OFFENDER_SENTENCES")
+// Warning: on mandatory fields mapped - just enough to create test entities
 data class OffenderSentence(
```

Comment:
no doubt we will fill in the rest some time later this year when we will actually create sentences in production